sysbuild/Kconfig.sysbuild

```diff
@@ -58,6 +58,14 @@ config BUILD_OUTPUT_HEX
 	  The name of this file can be customized with CONFIG_KERNEL_BIN_NAME.
 	  This will be applied to all target images.
 
+# TODO: NCSDK-28330
+# This check will fail for variants
+BOARD_NS_QUALIFIER_CHECK := $(substring,$(BOARD_QUALIFIERS),-3)
+
+config BOARD_IS_NON_SECURE
+	bool
+	default y if "$(BOARD_NS_QUALIFIER_CHECK)" = "/ns"
```

Comment:
Should be fine until we have a long term solution which accounts for https://github.com/zephyrproject-rtos/zephyr/issues/70721 and/or building for `nrf5340dk` (basename), once either is supported.